src/System.Reflection.DispatchProxy/ref/Configurations.props

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap10.0.15138;
```

Comment:
Why isn't the netstandard ref enough for this guy?

Reply:
I chatted with @ericstj and he said that it is fine to use netstandard as long as we are sure that we won't add any Apis that will cause a version bump on this guy anymore, but since we were not sure for the three libraries if this would be the case, we thought that it was better to have a TFM-specific asset.